Core/GameInstance.cs

```diff
@@ -241,6 +241,10 @@ public static string PortableDir(IGame game)
 
             // Find the directory our executable is stored in.
             string exeDir = Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location);
+			if (exeDir.Length == 0)
+			{
+				return null;
+			}
```

Comment:
```suggestion
            if (exeDir.Length == 0)
            {
                return null;
            }
```

Please keep the indentation consistent with the existing code.